tests/deep_inspection/test_standalone_di.py

```
@@ -49,3 +70,202 @@ def test_wait_for_completion(self) -> None:
+            raise AssertionError(
+                f"DI snapshot '{DI_SNAPSHOT_NAME}' still present on VM '{di_vm_name}' "
+                f"{DI_SNAPSHOT_CLEANUP_TIMEOUT}s after cancel. All snapshots: {remaining}"
+            )
+
+    def test_rerun_di_after_cancel(
+        self,
+        di_resolved_vm: dict[str, str],
+        di_connection_secret: Secret,
+        di_vddk_image: str,
+        fixture_store: dict[str, Any],
+        ocp_admin_client: "DynamicClient",
```

Ruff (0.15.20)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)